Fix mmcv version error

There was this error:

```
Exception: Error while executing python command '/var/www/vendor/biigle/maia/src/config/../resources/scripts/object-detection/TrainingRunner.py /var/www/storage/maia_jobs/maia-796-object-detection/input-training.json /var/www/storage/maia_jobs/maia-796-object-detection/output-dataset.json':\n
      Traceback (most recent call last):\n
        File "/var/www/vendor/biigle/maia/src/config/../resources/scripts/object-detection/TrainingRunner.py", line 9, in <module>\n
          from mmdet.utils import setup_cache_size_limit_of_dynamo\n
        File "/opt/conda/lib/python3.10/site-packages/mmdet/__init__.py", line 17, in <module>\n
          and mmcv_version < digit_version(mmcv_maximum_version)), \\n
      AssertionError: MMCV==2.1.0 is used but incompatible. Please install mmcv>=2.0.0rc4, <2.1.0.\n
       in /var/www/vendor/biigle/maia/src/Jobs/DetectionJob.php:143
```

It has failed job ID 36959 (MAIA job 796 (deleted)).

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,10 +1,10 @@
 torch==2.1.*
 torchvision==0.16.*
-mmcv==2.0.*
+mmcv==2.1.*
 # Update CUDA Version if necessary.
 # See: https://mmcv.readthedocs.io/en/latest/get_started/installation.html#install-with-pip
 -f https://download.openmmlab.com/mmcv/dist/cu117/torch2.0/index.html
-mmdet==3.1.*
+mmdet==3.2.*
 albumentations
 scikit-learn
 scikit-image

